keras/preprocessing/image.py

```diff
@@ -328,9 +352,16 @@ def load_img(path, grayscale=False, target_size=None):
         if img.mode != 'RGB':
             img = img.convert('RGB')
     if target_size:
-        hw_tuple = (target_size[1], target_size[0])
-        if img.size != hw_tuple:
-            img = img.resize(hw_tuple)
+        width_height_tuple = (target_size[1], target_size[0])
+        if img.size != width_height_tuple:
+            if interpolation not in _PIL_INTERPOLATION_METHODS:
```

Comment:
Please add a unit test for this feature, include a check for the ValueError.

Reply:
Are there any test images shipped with keras that could be used for this test? Otherwise, it is not possible to get to this point without a failure before getting to this point.

Reply:
We haves test code that creates test images and write them in a temporary folder. You can check out existing tests for `preprocessing/image`.